config/locales/server.en.yml

```diff
@@ -1759,6 +1759,14 @@ en:
 
       (If the link above has expired, choose "I forgot my password" when logging in with your email address.)
 
+  download_backup_mailer:
+    title: "Download Backup Mailer"
+    subject_template: "[%{site_name}] Backup Download"
+    text_body_template: |
+      Click this link to download your backup file: %{backup_file_path}
+    no_token: |
+      Sorry, that backup download link has already been used or has expired.
```

Comment:
Hmm probably `this backup link has already been used or has expired?` because the error message is shown when the user has visited the link.